tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow EDTF Date String."""
+
+import pytest
+from edtf.parser.grammar import ParseException, level0Expression
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import EDTFDateString
+
+
+def test_deserialize():
+
+    class MySchema(Schema):
+        f = EDTFDateString()
+
+    assert MySchema().load({'f': "2020-01-02garbage"}) == {'f': "2020-01-02"}
```

Comment:
minor: same than for the language tests, you might want to use `pytest.parameterized` to make it consistent, but is ok :)

Reply:
Changes done :)